sys/fmt/fmt.c

```diff
@@ -194,12 +194,12 @@ size_t fmt_u16_dec(char *out, uint16_t val)
 
 size_t fmt_s32_dec(char *out, int32_t val)
 {
-    int negative = (val < 0);
+    unsigned negative = (val < 0) ? 1 : 0;
```

Comment:
Looking at the side-by-side diff, does appending ```? 1 : 0``` really increase readability?

Reply:
IMO `(val < 0)` is perfectly readable and to the point.

Reply:
@gebart what do you think?

Reply:
> @gebart what do you think?

Just saw that you already replied. IMO if it is unclear which is better to read, let's lower the amount of characters that our poor brains need to parse.